optaplanner-build-parent/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.optaplanner</groupId>
+    <artifactId>optaplanner-parent</artifactId>
+    <version>8.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>optaplanner-build-parent</artifactId>
+  <packaging>pom</packaging>
```

Comment:
So, why this pom?
- The `optaplanner-parent` will (after step 3) hold all the metadata (url, scm, repo's, etc), maven plugin configuration (think enforcer, rules etc) etc. **But it cannot hold the dependencies because optaplanner-bom extends it.**
- The `optaplanner-build-parent` will (after step 3) extend optaplanner-parent and hold all dependencies, and also import optaplanner-bom.
- optaplanner-core, etc extend optaplanner-build-parent (not optaplanner-parent)

Reply:
If I understand it correctly, the `optaplanner-parent` should server as a parent pom for both the `optaplanner-bom` and the `optaplanner-build-parent`.

What is the reason for the intention to define plugin configuration in the `optaplanner-parent`? Plugin configuration is specific for the build of the OptaPlanner project, so I would rather expect them to be defined in the `optaplanner-build-parent`.

Reply:
Good point. Agreed. When we do step 3, plugin configuration should go into optaplanner-build-parent, not in optaplanner-parent.